python/tvm/relay/frontend/tflite.py

```diff
@@ -157,6 +157,8 @@ def __init__(self, model, subgraph, exp_tab):
             'UNPACK': self.convert_unpack,
             'WHERE': self.convert_select,
             'ZEROS_LIKE': self.convert_zeros_like,
+            'ARG_MIN': self.convert_arg_min,
+            'ARG_MAX': self.convert_arg_max,
```

Comment:
Please arrange alphabetically similar to other ops.

Reply:
Done